main/kotlin/com/example/people_managment_system/dtos/EmployeeDto.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.people_managment_system.dtos
+
+import java.io.Serializable
+
+data class EmployeeDto(
+    var employeeID: Int? = null,

```

Comment:
There are many fields in here that can be nullable upon a creation of an employee, I recommend making it more strict to ensure values get persisted for employees.

For example, the `employeId` is a crucial value, if this is null then it can cause problems.